src/cmcdKeys/common.js

```diff
@@ -1,39 +0,0 @@
-export const roundedToNearstHundredth = (number) => {

```

Comment:
CML will correctly round the bitrate values, so these functions are no longer needed.